packages/graphback-cli/src/commands/init.ts

```diff
@@ -0,0 +1,28 @@
+import yargs from 'yargs';
+import { init } from '../components/init'
+
+type Params = { name?: string, templateName?: string, templateUrl: string }
+
+export const command = 'init <name>'
+
+export const desc = 'Create Graphback project from available templates'
+
+// tslint:disable-next-line: typedef
+export const builder = (args: yargs.Argv) => {
+  args.positional('name', {
+    describe: 'Project name',
+    type: 'string',
+  })
+  args.option('templateName', {
+    describe: 'Name of the predefined template',
+    type: 'string'
+  })
+  args.option('templateUrl', {
+    describe: 'GitHub URL of the template. For example (http://github.com/wtrocki/graphback-hapijs#master)',
```

Comment:
http://github.com/wtrocki/graphback-hapijs#master is a 404, might be good to have a real link?

Reply:
There is no repo that will currently work with /template folder. 
I'm creating separate issue to sort that out

Reply:
I guess I will change `/template` to the `/`